Rpackage/R/mmit.cv.R

```diff
@@ -0,0 +1,47 @@
+mmit.cv <- structure(function(target.mat, feature.mat, 
+                              param_grid, feature_names = NULL, 
+                              n_folds = 3, scorer = NULL, loss = "hinge",
```

Comment:
Scorer is a function that is always increasing (e.g., zero one loss or -MSE).

Reply:
We should add attributes to our scoring functions so that we know if we should take the min or max. Also mention in the doc, that we expect the "direction" attribute for any user defined scoring function.

```
mse <- structure(mse <- function(y_true, y_pred){
+     error <- 0.0
+     for(i in 1 : length(y_pred)){
+         if(y_true[i,1] > y_pred[i]){
+             error <- error + (y_true[i,1] - y_pred[i])**2
+         }
+         else if(y_true[i,2] <= y_pred[i]){
+             error <- error + (y_true[i,2] - y_pred[i])**2
+         }
+     }
+     return((error / length(y_pred)))
+ }, direction=min)
```

Reply:
We also decided to add "worst" attribute which will be used in fit and score.

Reply:
done